examples/1_Spectra_handling/Kurucz_partition_functons_potential_lowering.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+"""
+========================
+Kurucz partition functions and potential lowering
+========================
+
+For atomic species in the Kurucz databank, some species come with dedicated partition function tables provided, which depend on both temperature and potential lowering. The `potential_lowering` parameter of :py:class:`~radis.lbl.factory.SpectrumFactory` provides a way for the user to set this value to make use of these tables in the interpolator.
+
+When such tables are unavailable or `potential_lowering` is `None` (the default), the partition functions are instead interpolated from [Barklem & Collet 2016]_ Table 8.
+
+The value can thereafter be changed on the fly by changing the `sf.input.potential_lowering` attribute of the :py:class:`~radis.lbl.factory.SpectrumFactory` instance ``sf``, the result of which is reflected the next time partition function interpolator's `._at` method is used, without any need to re-initialise it.
+
+Allowable values are usually: -500, -1000, -2000, -4000, -8000, -16000, -32000. This is based on what's been encountered in the partition function tables of the species checked so far, but documentation of the Kurucz linelists is unclear on whether this is the case for all species.
+
+The temperature ranges of the partition functions from [Barklem & Collet 2016]_ may differ from those of the dedicated Kurucz tables, so where you can calculate spectra at a particular temperature with the Kurucz tables:
```

Comment:
Kurucz temperature values seem to typically start at 100 actually, hence the choice of 99 in the example

Reply:
I think typical lower bound of temperature range for Kurucz pfs is actually 100. Implication for allowable values of potential_lowering was if you want to set it to 'something' rather than 'None' (default as mentioned).